tests/test_sap_success_factors/test_learner_data.py

```diff
@@ -302,3 +338,61 @@ def get_course_grade(course_id, username):
         assert learner_data[2].course_completed
         assert learner_data[2].completed_timestamp == self.NOW_TIMESTAMP
         assert learner_data[2].grade == grade
+
+    @ddt.data(
+        (True, True, 'audit', 1), (True, True, 'verified', 1),
+        (True, False, 'audit', 0), (True, False, 'verified', 1),
+        (False, True, 'audit', 0), (False, True, 'verified', 1),
+        (False, False, 'audit', 0), (False, False, 'verified', 1),
```

Comment:
Yep, sure. At first I had it as just 3 parameters (bool, bool, str) so it made a little more sense to have it this way since everything was equal in the first 2 parameters for each side except the string ('audit' vs 'verified'), but now with the numbers it makes sense to make it 1 column.